.gitignore

```diff
@@ -25,3 +25,8 @@ npm-debug.log
 # WebStorm user-specific
 .idea/workspace.xml
 .idea/tasks.xml
+.idea/cesium.iml
+.idea/encodings.xml
+.idea/inspectionProfiles/Project_Default.xml
+.idea/jsLinters/jshint.xml
+.idea/misc.xml
```

Comment:
Thanks for the hint!
My Webstorm seemed to have changed some of them automatically, and I didn't want to commit those changes (but then I don't need to add the changes to .gitignore, of course). I wanted to revert this inside the pull request, but didn't do it so far.